lib/commands/command-update.bash

```diff
@@ -26,26 +26,19 @@ do_update() {
     printf "Updated asdf to latest on the master branch\n"
   else
     # Update to latest release
+    local sha_of_tag
+    local tag
+
+    # fetch tags from remote
     git fetch origin --tags || exit 1
 
-    if [ "$(get_asdf_config_value "use_release_candidates")" = "yes" ]; then
-      # Use the latest tag whether or not it is an RC
-      tag=$(git tag | sort_versions | sed '$!d') || exit 1
-    else
-      # Exclude RC tags when selecting latest tag
-      tag=$(git tag | sort_versions | grep -vi "rc" | sed '$!d') || exit 1
-    fi
+    sha_of_tag=$(git rev-list --tags --max-count=1) || exit 1
```

Comment:
Is this `git rev-list` command returning the most recently created tag, or the tag with the highest semantic version? 

If this is returning the most recently created tag, we may want to stick with the `sort_versions` approach here in case we ever tag a new patch version for old minor version of asdf. I know we've not done that thus far, so maybe it's not necessary.

Reply:
Yeah, looking at this again I'd suggest keeping `sort_versions` for now. All other changes look good to me.

Reply:
I have a different solution to propose here which would still remove `sort_versions`, I have just been busy with work so haven't had a chance to write it up. I would like to discuss that alternative before keeping `sort_versions`